content/introduction/_index.md

```diff
@@ -6,16 +6,18 @@ disableToc: false
 ---
 
 ## Introduction
+
```

Comment:
Just trying to understand why the newline here and spaces throughout the document have been added. 
Was it done by a prettifier plugin?

Reply:
Yes, the newlines and spaces were added by a prettifier plugin automatically, and I wasn't aware of it. I'll ensure it's disabled to maintain the original formatting.